src/imitation/policies/interactive.py

```diff
@@ -44,34 +66,35 @@ def _choose_action(self, obs: np.ndarray) -> np.ndarray:
         return np.array([self.action_key_to_index[key]])
 
     def _get_input_key(self) -> str:
+        """Obtains input key for action selection."""
         print(
             "Please select an action. Possible choices in [ACTION_NAME:KEY] format:",
-            ", ".join(
-                [f"{n}:{k}" for n, k in zip(self.action_names, self.action_keys)]
-            ),
+            ", ".join([f"{n}:{k}" for k, n in self.action_keys_names.items()]),
         )
 
         key = input("Your choice (enter key):")
-        while key not in self.action_keys:
+        while key not in self.action_keys_names.keys():
             key = input("Invalid key, please try again! Your choice (enter key):")
 
         return key
 
     @abc.abstractmethod
-    def _render(self, obs: np.ndarray) -> Optional[object]:
-        """Renders an observation, optionally returns a context object for later cleanup."""
+    def _render(self, obs: np.ndarray) -> typing.Optional[object]:
+        """Renders an observation, optionally returns a context for later cleanup."""
 
     def _clean_up(self, context: object) -> None:
         """Cleans up after the input has been captured, e.g. stops showing the image."""
         pass
```

Comment:
```suggestion
```
Not needed given there's a docstring. Either remove `pass` or add `pass` to `_render` for consistency.